lib/sycamore/sycamore/tests/integration/connectors/neo4j/test_docset_to_neo4j.py

```diff
@@ -107,8 +18,7 @@ def children_to_section(doc: HierarchicalDocument) -> HierarchicalDocument:
             partitioner=SycamorePartitioner(extract_table_structure=True, use_ocr=True, extract_images=True),
             num_gpus=0.2,
         )
-        .map(restructure_doc)
-        .map(children_to_section)
+        .extract_graph_structure(extractors=[])
```

Comment:
what is this actually writing to neo4j? also is the AUTH the default vals neo4j local uses?

Reply:
This is writing to neo4j the layout of the ingested document(a page of ray). It mostly tests if the writer is following expected behavior(as in, treating each document after explode as a node with a bunch of relationships).

The auth is the one i was using locally because using the default password at the time was causing some issues cause i tried to set the default password manually but it was too short for neo4j so it just died. I'll add it back.

Reply:
So turns out the default neo4j password cant be used to access the database via the python driver. What they do is make you login through the UI and have them choose a new password if the auth is not specified in the docker params. I just decided to use no auth for the github actions and the code has updated to that version now.